### PR TITLE
J-263: Fix popular podcasters markup

### PR DESCRIPTION
**Pull request by EugeniusGR**
image

### CODE REVIEW

File: frontend/src/components/homepage/components/popular-users/popular-users.tsx [hunk 1]
```diff
@@ -1,22 +1,35 @@
...
+        </ul>
+      ) : (
+        <ul className={styles.usersRow}>
+          {popularUsers.map((user) => (
+            <>
+              <PopularUser key={user.id} user={user} />
+            </>
+          ))}
+        </ul>
+      )}
```
Comment (Denidzio) [inline frontend/src/components/homepage/components/popular-users/popular-users.tsx]:
```suggestion
         <ul className={getAllowedClasses(styles.usersRow, popularUsers.length >=5 && styles.groupPodcastes)}>
          {popularUsers.map((user) => (
            <>
              <PopularUser key={user.id} user={user} />
            </>
          ))}
        </ul>
```

**Review by Denidzio — CHANGES_REQUESTED** [review]

File: frontend/src/components/homepage/components/popular-users/popular-users.tsx [hunk 2]
```diff
@@ -1,20 +1,21 @@
...
   return (
     <div className={styles.root}>
       <h2 className={styles.title}>Popular podcaster</h2>
-      <ul className={styles.usersRow}>
+      <ul className={getAllowedClasses(styles.usersRow, popularUsers.length >=5 && styles.groupPodcastes)}>
         {popularUsers.map((user) => (
-          <PopularUser key={user.id} user={user} />
+          <>
+            <PopularUser key={user.id} user={user} />
+          </>
```
Comment (Denidzio) [inline frontend/src/components/homepage/components/popular-users/popular-users.tsx]: Why did you wrap it by React.Fragment?

Comment (Denidzio) [reply frontend/src/components/homepage/components/popular-users/popular-users.tsx]: I think it will cause error with 'key' prop. Check it in the console

Comment (EugeniusGR) [reply frontend/src/components/homepage/components/popular-users/popular-users.tsx]: oh, I did some checks with count of users and forget to delete it, thx ;)

File: frontend/src/components/homepage/components/popular-users/popular-users.tsx [hunk 3]
```diff
@@ -1,18 +1,22 @@
...
 const PopularUsers: React.FC<Props> = ({ popularUsers }: Props) => {
-
   return (
     <div className={styles.root}>
       <h2 className={styles.title}>Popular podcaster</h2>
-      <ul className={styles.usersRow}>
+      <ul
+        className={getAllowedClasses(
+          styles.usersRow,
+          popularUsers.length >= 5 && styles.groupPodcastes,
```
Comment (Denidzio) [inline frontend/src/components/homepage/components/popular-users/popular-users.tsx]: What does it mean 5? It should be a constant

**Review by Denidzio — CHANGES_REQUESTED** [review]

File: frontend/src/components/homepage/components/popular-users/styles.module.scss [hunk 4]
```diff
@@ -1,32 +1,40 @@
...
+  padding: 0;
+  grid-gap: 35px;
+  overflow-x: auto;
 
-    li {
-      margin-bottom: 15px;
-    }
+  li {
+    margin-bottom: 15px;
+    margin-right: 35px;
```
Comment (Denidzio) [inline frontend/src/components/homepage/components/popular-users/styles.module.scss]: You have used `grid-gap` already. Why did you start to use `margin-right`?

image

**Review by Denidzio — CHANGES_REQUESTED** [review]

File: frontend/src/components/homepage/components/popular-users/popular-users.tsx (hunk 3 above)
Comment (what1s1ove) [reply frontend/src/components/homepage/components/popular-users/popular-users.tsx]: agreed

**Review by what1s1ove — CHANGES_REQUESTED** [review]

File: frontend/src/components/homepage/components/popular-users/styles.module.scss (hunk 4 above)
Comment (what1s1ove) [reply frontend/src/components/homepage/components/popular-users/styles.module.scss]: agreed

File: frontend/src/components/homepage/components/popular-users/popular-users.tsx [hunk 5]
```diff
@@ -1,18 +1,24 @@
...
+import { getAllowedClasses } from 'helpers/helpers';
 import styles from './styles.module.scss';
 
 type Props = {
-  popularUsers: User[],
+  popularUsers: User[];
 };
 
-const PopularUsers: React.FC<Props> = ({ popularUsers }: Props) => {
+const MAXIMUM_PODCASTERS_WITHOUT_MARKUP = 5;
```
Comment (what1s1ove) [inline frontend/src/components/homepage/components/popular-users/popular-users.tsx]: move to `frontend/src/components/homepage/components/popular-users/common/constants.ts`

**Review by what1s1ove — CHANGES_REQUESTED** [review]

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**